corehq/apps/users/migrations/0065_transfer_to_tableau_config_permission.py

```diff
@@ -0,0 +1,73 @@
+import sys
+import traceback
+
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+from corehq.toggles import TABLEAU_USER_SYNCING
+from corehq.apps.domain_migration_flags.api import (
+    ALL_DOMAINS,
+    get_migration_status,
+    set_migration_complete,
+    set_migration_started,
+)
+from corehq.apps.domain_migration_flags.exceptions import (
+    DomainMigrationProgressError,
+)
+from corehq.apps.domain_migration_flags.models import MigrationStatus
+
+
+MIGRATION_SLUG = "mirror_web_users_permissions_to_tableau_config_permissions"
+
+
+@skip_on_fresh_install
+def _assert_migrated(apps, schema_editor):
+    UserRole = apps.get_model('users', 'UserRole')
+    Permission = apps.get_model('users', 'Permission')
+
+    def transfer_web_user_permission_to_tableau_config_permission():
+        status = get_migration_status(ALL_DOMAINS, MIGRATION_SLUG)
+        if status == MigrationStatus.COMPLETE:
```

Comment:
I think it would still be helpful for logging purposes in case the migration fails for some domains. What do you mean by "migration is restartable"?  Do you mean if the migration logic was in a management command that could be rerun?

Reply:
If there is an exception, some domains would have already been migrated and others would not have (I'm pretty sure that's right?). In that situation, tracking the status would record in `DomainMigrationProgress` model which domains have and haven't been completed yet. This would be more fidelity than just the overall Django migration status. 

I don't think it matters much here since there's only a handful of domains. But I think that's what the additional purpose would be.